Formula/grib-api.rb

```diff
@@ -25,6 +27,7 @@ def install
     mkdir "build" do
       args = std_cmake_args
       args << "-DBUILD_SHARED_LIBS=OFF" if build.with? "static"
+      args << "-DPNG_PNG_INCLUDE_DIR=#{HOMEBREW_PREFIX}/include" << "-DENABLE_PNG=ON" if build.with? "libpng"
```

Comment:
Instead of `#{HOMEBREW_PREFIX}/include` should use `#{Formula["libpng"].opt_include}`